### DISCUSSION TITLE
Possibility to encrypt message without recipient key id

### DISCUSSION THREAD

**Discussion by martgil** [openpgpjs/openpgpjs#1572] [Q&A] [answered]
Hello, dear OpenPGP.js team,

I take a quick glance into the openpgp.js docs and I didn't see a thread related to hiding the recipient key id on message encryption. I wonder if openpgp.js has this option or if may have any plans to add it in the future?

Thanks again for making/maintaining Openpgp.js :)

**Comment by twiss** [comment]
Hello :wave: It's currently not implemented, and we don't have active plans for it, but are not opposed to it. If you're interested in implementing this, we can discuss about sketching a potential API for it :)

**Reply by martgil** [reply]
@twiss Thanks for welcoming me to contribute with OpenPGP.js. I wish I could participate but I'm not good at JS yet. Thanks for answering my question and have a great a day! :)

**Comment by twiss** [comment, marked as answer]
Hello :wave: We do actually have an option for this in the `encrypt` function, called `wildcard` (see https://docs.openpgpjs.org/global.html#encrypt). I forgot about that, apologies!

**Reply by martgil** [reply]
Hi - thank you very much. This is super helpful. Thank you and take care :)
